src/main/java/datawave/query/model/ModelKeyParser.java

```diff
@@ -51,12 +84,19 @@ public static FieldMapping parseKey(Key key, Value value) {
         // we can have attributes no matter the mapping
         attributes.addAll(Arrays.asList(StringUtils.split(new String(value.get(), StandardCharsets.UTF_8), ',')));
         
-        if (1 == colq.length) {
-            // in this case we expect model field attributes (not specific to a mapping)
+        if (0 == colq.length || (colq.length == 1 && colq[0].isEmpty())) {
```

Comment:
nitpick, but 0 == colq.length will never happen. also i'm surprised that ModelKeyParser.parseKey(VERSION_KEY1, VERSION_VALUE1) doesn't throw IAE where CQ is an empty string.

Reply:
OK, This is weird, but that test case actually fails in my IDE....   Not convinced the tests below actually did their job.... DOH!